src/Components/BookPage/BookCover.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import {
+  Typography, Chip, Box,
+} from '@mui/material';
+import Logo from '../../Assets/Images/TCK PNG Logo.png';
+
+const styles = {
+  bookCover: {
+    maxWidth: '20vw',
+    filter: 'drop-shadow(4px 4px 50px rgba(0, 0, 0, 0.25))',
+    marginRight: '2vw',
+    '@media (max-width: 960px)': {
+      maxWidth: '90vw',
+      margin: 'auto',
+    },
+  },
+  topDownContainer: {
+    display: 'block',
+    margin: 'auto 1vw auto 1vw',
+  },
+  tagHeader: {
+    textAlign: 'left',
+    fontFamily: 'Work Sans',
+    fontSize: '.9em',
+    fontWeight: '600',
+    margin: '2vh auto 1vh auto',
+    '@media (max-width: 960px)': {
+      width: '90%',
+      margin: '2vh auto 1vh auto',
+    },
+  },
+  tagContainer: {
+    overflowWrap: 'break-words',
+    margin: 'auto',
+    textAlign: 'left',
+    '@media (max-width: 960px)': {
+      width: '90%',
+    },
+  },
+  chip: {
+    margin: '0.2vw',
+    fontSize: '0.9em',
+    fontFamily: 'DM Sans',
+  },
+
+};
+
+const chipColors = [
+  { // red  -- race/ethnicity
+    color: '#A51717',
+    backgroundColor: '#FFDFDF',
+  },
+  { // green  -- genre
+    color: '#004F4A',
+    backgroundColor: '#D5EFF1',
+  },
+  { // yellow -- identity
+    color: '#6F4323',
+    backgroundColor: '#FAE9D0',
+  },
+  { // blue -- themes/lessons
+    color: '#004A90',
+    backgroundColor: '#D0E6FB',
+  },
+  { // purple -- religion
+    color: '#131C72',
+    backgroundColor: '#E2E5FF',
+  },
+];
+
+/* Tags
+ */
+const createTags = (identityTags, raceEthnicity, genre, themesLessons, religion) => {
```

Comment:
Each function/component should be in their own file! This is basically a component, so can you move it to a BookTag file instead of keeping it on the BookCover file?